corehq/apps/app_manager/translations.py

```diff
@@ -446,6 +452,10 @@ def update_form_translations(sheet, rows, missing_cols, app):
     :return:  Returns a list of message tuples. The first item in each tuple is
     a function like django.contrib.messages.error, and the second is a string.
     """
+    def value_is_safe(value_element):
+        is_safe = lambda el: isinstance(el, etree.Entity) or isinstance(el, etree.Element) and el.tag in SAFE_TAGS
```

Comment:
These aren't actually classes. We use lxml here which uses `etree._Entity` and `etree._Element`. I'm not 100% sure on the differences between those